parm/config/gfs/config.resources

```diff
@@ -381,6 +381,7 @@ elif [[ "${step}" = "ocnanalrun" ]]; then
    case ${CASE} in
       C384)
         npes=480
+        export memory_ocnanalchkpt="2.8TB"
```

Comment:
oof!

Reply:
30 offline members for the EnVAR @aerorahul .

Reply:
@guillaumevernieres and @aerorahul , should we set `memory_ocnanalrun` instead of `memory_ocnanlchkpt`?  Line 384 is in an `ocnanalrun` block.

Reply:
> @guillaumevernieres and @aerorahul , should we set `memory_ocnanalrun` instead of `memory_ocnanlchkpt`? Line 384 is in an `ocnanalrun` block.

That's a snafu on my side. I'll issue a fix later, there are other things that weren't merged properly too.